src/Domain/CoordinateMaps/BulgedCube.hpp

```diff
@@ -431,15 +434,20 @@ namespace CoordinateMaps {
  * \end{bmatrix}
  * \f]
  *
- * In the case where the same coordinates are used for the cube and the sphere,
- * we have \f$\xi = \Xi\f$, etc. In this case, the formula reduces further.
- * It is given by:
+ * In the above formula, the lower case Greek coordinates are artifacts of
```

Comment:
The word `artifacts` is confusing to me. I also don't understand linear/nonlinear interpolation because the difference appears to me to be a choice of coordinates and not the linearity of the interpolation.  And there are 6 bounding surfaces, not 2, right? What about:

"In the above formula, the lower case Greek coordinates appear because we explicitly chose linear interpolation in the equidistant coordinates for interpolating between the bounding surfaces.  The upper case Greek coordinates can be specified as either the equidistant or equiangular coordinates.  If we choose the upper case Greek coordinates to be equidistant coordinates, so that \f$\xi = \Xi\f$, etc, then the formula reduces further."

Is the above correct, or am I still misunderstanding something?  (In particular, can you choose *both* the lower case and upper case Greek coordinates to be equiangular? I am assuming not, but if you can choose the lower case Greek coords to be equiangular, then there needs to be something said about how \xi, \eta, \zeta in the interpolation formula above are not necessarily the equidistant coordinates.)

Reply:
Right, both the upper and lower greek letters can be either equiangular or equidistant, and the simplified formula is when the two match. My intention was to have lower case greek coordinates only be equidistant though, so I will need to find a way to represent that both the coordinates used for the surfaces and for the interpolation can be independently one of two choices.